frontend/packages/tools/devBuild.mjs

```diff
@@ -0,0 +1,32 @@
+import esbuild from "esbuild";
+import fs from "fs";
+import path from "path";
+
+const args = process.argv.slice(2);
```

Comment:
whats the `slice(2)` for?

Reply:
the second command line argument (e.g. in this case it's the source directory)